src/pages/NewChatPage.js

```diff
@@ -235,16 +235,16 @@ class NewChatPage extends Component {
             maxParticipantsReached,
         );
         return (
-            <ScreenWrapper>
-                {({didScreenTransitionEnd}) => (
+            <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+                {({didScreenTransitionEnd, safeAreaPaddingBottomStyle}) => (
```

Comment:
Why does this need the `safeAreaPaddingBottomStyle` if `includeSafeAreaPaddingBottom` is false?

Reply:
We show a button at the bottom of the screen when some users are selected. Otherwise, we don't need any padding at all.